py_mmd_tools/nc_to_mmd.py

```diff
@@ -629,13 +643,16 @@ def to_mmd(self, *args, **kwargs):
         fchecksum = md5hasher.hash_file(file_for_checksum_calculation)
         self.metadata['storage_information'] = {
                 'file_name': os.path.basename(self.netcdf_product),
-                'file_location': os.path.abspath(self.netcdf_product).replace(os.path.basename(self.netcdf_product),''),
+                'file_location': self.netcdf_product.replace(os.path.basename(self.netcdf_product),''),
```

Comment:
Why are you using replace here?

Is it not like `os.path.dirname(self.netcdf_product)`

Reply:
I think that will include the filename.. I tried a bit back and forth and concluded this worked